### DISCUSSION TITLE
On my windows 10 installation it crashes as soon as I create a new document

### DISCUSSION THREAD

**Discussion by wsteffe** [dune3d/dune3d#77] [Q&A] [answered]

**Comment by wsteffe** [comment]
To be more precise it crashed on windows 10 inside embadded in a vmware virtual machine. After this experiment I have installed dune3d on a normal windows 10 system (running on bare metal) and there it didn't crash.

**Comment by carrotIndustries** [comment, marked as answer]
Might be due to OpenGL problems. Dune 3D requires OpenGL 4 or so and most VMs don't provide that. If you run it from an MINGW shelll, you should get some more debug output.

**Comment by wsteffe** [comment]
Ok thanks.
